src/tests/ftest/pool/simple_create_delete_test.yaml

```diff
@@ -6,27 +6,11 @@ hosts:
 server_config:
  name: daos_server
 tests:
- modes: !mux
+ modes:
   modeall:
    mode:
     - 511
     - PASS
-  readonly:
-   mode:
-    - 73
-    - FAIL
-  writeonly:
-   mode:
-    - 146
-    - PASS
-  execonly:
-   mode:
-    - 292
-    - PASS
-  modebogus:
```

Comment:
Mode itself is obsolete. No matter what type/value you pass it's going to pass. I'm just keeping one mode there because the API has mode in it's signature and without that the test would fail.

Reply:
Thanks for the clarification. But if 'mode' is obsolete, doesn't that make this entire test obsolete?

Reply:
I guess my question is what is the purpose of this test since the original intention was to test various supported modes (which is now obsolete)?

Reply:
Mode is just one variant of this test, there are other variants like UID, GID etc. which are still valid. I was also told that the pool creation is going to be through command line as opposed to calling the API in the future. At that time, I believe we would reconsider whether to keep this test or not or even tweak it to adopt to new way of pool creation.

Reply:
Ok, nvm, i think i understand the situation now. You are removing the redundant test scenarios which is no longer supported for a create/delete test.
Cool, thanks!